lib/grinder_tools.dart

```diff
@@ -175,6 +176,58 @@ class Tests {
   }
 }
 
+/// A wrapper around the `test` package. This class is used to run your unit
+/// tests.
+class Test {
```

Comment:
I like that `Test` matches the name of the package (`test`), but without reading the documentation for it, I'd assume `Test` represents an individual unit test, or run thereof, not a test running API.  Also once https://github.com/dart-lang/test/issues/23 is fixed, that will expose just such a `Test` class, and that may become the preferred way to run tests from grinder since it avoid shelling out, but we would likely have a name conflict.  Some other suggestions would be `Tester`, `TestRunner`, or `TestApp`.  @zoechi thoughts?